docs/configuration/multipledomains.md

```diff
@@ -64,6 +64,92 @@ Multiple top-level domains may also be setup by following the instructions below
          - sub1.alternate2.test:${TRAEFIK_ADDRESS:-0.0.0.0}
          - sub2.alternate2.test:${TRAEFIK_ADDRESS:-0.0.0.0}
     ```
+### Magento 1 Subfolder setup
+If you have a M1 subfolder setup to run your multiple stores, you will have to extend the `nginx` configuration. To do that, add `.warden/warden-env.yml`
+file with the following content to your project root:
+```
+version: "3.5"
+services:
+  nginx:
+    volumes:
+      - ./.warden/nginx/custom.conf:/etc/nginx/default.d/custom.conf
```

Comment:
Are you sure this works with `custom.conf`? Aren't the config files read alphabetically, so that we need to make sure that the custom file is read after `magento1.conf`?

Reply:
I tried it, and it worked, but a good point anyway.
Just checked `images/nginx/context/etc/nginx/conf.d/default.conf` again and found the following:

```
    include /etc/nginx/available.d/${NGINX_TEMPLATE};
    include /etc/nginx/default.d/*.conf;
```
So we're definitely loaded after `magento1.conf`, so this should be fine

Reply:
@sprankhub the application config files are loaded from `/etc/nginx/available.d`, before loading files from `/etc/nginx/conf.d`.